edgelet/iotedge/src/support_bundle.rs

```diff
@@ -157,6 +165,134 @@ where
                 })
             })
     }
+
+    fn write_edgelet_log_to_file(mut state: BundleState<M>) -> Result<BundleState<M>, Error> {
+        println!("Getting system logs for iotedge");
+        let since_time: DateTime<Utc> = DateTime::from_utc(
+            NaiveDateTime::from_timestamp(state.log_options.since().into(), 0),
+            Utc,
+        );
+        let since = since_time.format("%F %T").to_string();
+
+        #[cfg(unix)]
+        let inspect = ShellCommand::new("journalctl")
+            .args(&["-u", "iotedge"])
+            .args(&["-S", &since])
+            .arg("--no-pager")
+            .output();
+
+        #[cfg(windows)]
+        let inspect = ShellCommand::new("Get-WinEvent")
```

Comment:
Are you sure this works? `Get-WinEvent` is a PS commandlet. You'd need to spawn `powershell.exe` with argv `["-NoProfile", "-Command", "Get-WinEvent -ea ..."]`